app/controllers/SwitchesController.scala

```diff
@@ -3,7 +3,8 @@ package controllers
 import com.gu.googleauth.AuthAction
 import models.SupportFrontendSwitches
 import play.api.mvc.{AnyContent, ControllerComponents}
-import io.circe.generic.auto._
```

Comment:
👍 not providing our own decoder/encoder was having some weird effects when the model was changed